graders/python/requirements.txt

```diff
@@ -1,28 +1,28 @@
 beautifulsoup4==4.12.3
-bokeh==3.4.1
+bokeh==3.4.2
 colormath==3.0.0
 defusedxml==0.7.1
-Faker==25.3.0
-folium==0.16.0
-ipython==8.25.0
+Faker==26.0.0
```

Comment:
The Faker folks looooove their major versions. This one looks innocuous: https://github.com/joke2k/faker/blob/refs/tags/v26.0.0/CHANGELOG.md

Reply:
I think every major version bump from them has been non-breaking thus far.